main/src/ca/uwaterloo/flix/language/phase/ControlFlowAnalysis.scala

```diff
@@ -0,0 +1,1787 @@
+        case Expression.ProcessSpawn(exp, tpe, loc) => visitExp(result, exp)
+
+        //
+        // ProcessSleep.
+        //
+        case Expression.ProcessSleep(exp, tpe, loc) => visitExp(result, exp)
+
+        //
+        // ProcessPanic.
+        //
+        case Expression.ProcessPanic(msg, tpe, loc) => result
+
+        //
+        // ConstraintSet.
+        //
+        case Expression.FixpointConstraintSet(cs0, tpe, loc) => result
+
+        //
+        // Constraint Union.
+        //
+        case Expression.FixpointCompose(exp1, exp2, tpe, loc) =>
+          val e1 = visitExp(result, exp1)
+          val e2 = visitExp(e1, exp2)
+          e2
+
+        //
+        // Fixpoint Solve.
+        //
+        case Expression.FixpointSolve(exp, stf, tpe, loc) => visitExp(result, exp)
+
+        //
+        // Fixpoint Project.
+        //
+        case Expression.FixpointProject(sym, exp, tpe, loc) => visitExp(result, exp)
+
+        //
+        // Fixpoint Entails.
+        //
+        case Expression.FixpointEntails(exp1, exp2, tpe, loc) =>
+          val e1 = visitExp(result, exp1)
+          val e2 = visitExp(e1, exp2)
+          e2
+
+        //
+        // Fixpoint Fold.
+        //
+        case Expression.FixpointFold(sym, exp1, exp2, exp3, tpe, loc) =>
+          val e1 = visitExp(result, exp1)
+          val e2 = visitExp(e1, exp2)
+          val e3 = visitExp(e2, exp3)
+          e3
+
+        //
+        // Error Expressions.
+        //
+        case Expression.HoleError(sym, tpe, loc) => result
+        case Expression.MatchError(tpe, loc) => result
+        case Expression.SwitchError(tpe, loc) => result
+
+        //
+        // Unexpected Expressions.
+        //
+        case Expression.LambdaClosure(fparams, freeVars, _, tpe, loc) => throw InternalCompilerException(s"Unexpected expression: '${lExp.exp.getClass}'.")
+        case Expression.Lambda(args, body, tpe, loc) => throw InternalCompilerException(s"Unexpected expression: '${lExp.exp.getClass}'.")
+        case Expression.Apply(_, args, tpe, loc) => throw InternalCompilerException(s"Unexpected expression: '${lExp.exp.getClass}'.")
+      }
+    }
+
+    val (varFlow, expFlow) = labeledDefs.foldLeft((Map(), Map()): (AbstractEnvironment, AbstractCache)) {
+      case (acc, (_, defn)) => visitExp(acc, defn.exp)
+    }
+
+    def inlineClosApp(lExp: LabeledExpression, tpe: Type, loc: SourceLocation): Option[Symbol.DefnSym] = {
+      val closures = lExp.exp match {
+        case Expression.Var(sym, tpe, loc) => varFlow.getOrElse(sym, Set())
+        case _ => expFlow.getOrElse(lExp, Set())
+      }
+      if (closures.size == 1) {
+        val clos = closures.head
+        if (clos.freeVars.isEmpty) {
+          val defn = root.defs(clos.sym)
+          Some(defn.sym)
+        } else {
+          None
+        }
+      } else {
+        None
+      }
+    }
+
+    def inliner(exp0: SimplifiedAst.Expression) : SimplifiedAst.Expression =
+      exp0 match {
+        //
```

Comment:
Just drop these comments:

///
///
///

I know they are used in some other places, but we are trying to get rid of them.